README.md

```diff
@@ -20,11 +20,10 @@ MyDiary is an online journal where users can pen down their thoughts and feeling
 
 ## Technologies
 
-- Nodejs
-- Express
-- Mocha (with Chai as assertion library)
-- Babel (ES6+ to ES5 transpilation)
-- ESLint (with Airbnb style guide)
+- Nodejs (with ExpressJS)
```

Comment:
NodeJS is not a technology, it is a runtime engine. TDD is a methodology, ES6 is a language specification and Linting is validating a code through a set of rules. 

The heading can be `Tools and Technologies`, then TDD, linting, and Es6 should be removed. Tools can be the database used, the library used (such as react), the engine used ( Nodejs, .net, Laravel), the language (PHP, Java, Js, Scala, Ruby), cloud services etc

Reply:
Okay. That's revealing. I will make ammendments.

Reply:
I understand better now. Thanks.